### ISSUE TITLE
CSS Selection doesn't affect the feed content.

### ISSUE THREAD

**Issue by mikeymop** [FreshRSS/FreshRSS#3322]
I added an RSS Feed from TheVerge

`https://www.theverge.com/rss/index.xml`

It appears I have a valid selection at:
```css
.l-col__main
```
I test this by pressing the preview icon and it shows the entire article contents.

When I confirm, the feed contents still shows the truncated content from the RSS feed. Whether I refresh the feed or erase and create a new one this seems to always be the case.

Is there something special I have to set in FreshRSS in order to have it show the full article contents by default?

Note, I am using Readably as a fever client and it also receives the truncated content. However in Readably, when I click 'Continue Reading' it is able to parse the original page and get the article in the app. Where as FreshRSS opens the actual page which I what i would like to avoid.

**Comment by Frenzie** [comment]
It works for me, but it's not completely clear to me if you mean the `Reload articles` button at the bottom by reloading the feed.

![Screenshot_2021-01-03_09-26-59](https://user-images.githubusercontent.com/202757/103474612-f7f67980-4da5-11eb-85d2-b9ea4e2d0bde.png)



<img src=https://user-images.githubusercontent.com/202757/103474568-74d52380-4da5-11eb-8b61-ea2a1c6cf3e4.png width=45%> <img src=https://user-images.githubusercontent.com/202757/103474565-743c8d00-4da5-11eb-955a-d0bf87788659.png width=45%>

**Comment by mikeymop** [comment]
My apologies, I realize that I was actually clicking 'Actualize' from the main view and wrongly assumed they were comparable.

I did ultimately get this working on *some* of my feeds, I noticed that if I left the article open long enough ~5 seconds on some feeds it did ultimately resolve with the full article.

Following your suggestion, I just tried reloading the articles for TheVerge and ArsTechnica. This natually took a while, given my hardware, however it worked for TheVerge, thanks!

Another example is ArsTechnica, and a few other feeds I use, have a 'read more' link in their feed that when clicked in Readably loads the article in the app but in FreshRSS this opens a browser page to the actual site.

It appears that even if I select the full article using the CSS FreshRSS still defaults to the truncated feed.

eg:
Feed: `http://feeds.arstechnica.com/arstechnica/index`
Selector: `#article-guts .article-content > p`

There is a 'Read 4 remaining paragraphs' link at the bottom.
<img width="662" alt="Screen Shot 2021-01-04 at 1 25 52 PM" src="https://user-images.githubusercontent.com/4002969/103566674-66b00180-4e90-11eb-919a-3d7d6d0542cf.png">

Is there something I may be missing for feeds that are expandable such as this?
